Sources/HTTP/Response/ResponseRepresentable.swift

```diff
@@ -28,3 +30,14 @@ extension Swift.String: ResponseRepresentable {
         )
     }
 }
+
+///Allows Foundation Data to be returned through closures
+extension Foundation.Data: ResponseRepresentable {
+    public func makeResponse() -> Response {
+        return Response(
+            status: .ok,
+            headers: [:],
```

Comment:
Thanks for reviewing, you're right headers shouldn't be specifically set here. I've removed `[:]` - the parameter is optional and it's `[:]` by default anyway, so this won't alter how things currently work.

PS: I was also thinking of something like instead of `makeResponse()`, making something like `makeResponse(headers: [HeaderKey: String]? = [:])`, but I'm not sure the design guide here so I'll leave it for later modifications.